lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -17037,3 +17037,24 @@ msgstr "Supprimer des collecteurs"
 
 msgid "Deploy configuration"
 msgstr "Déployer la configuration"
+
+msgid "Invalid endpoint type specified"
+msgstr "Type d'endpoint spécifié invalide"
+
+msgid "Invalid endpoint URL specified"
+msgstr "URL d'endpoint spécifié invalide"
+
+msgid "Request for authentication conditions custom endpoint has failed"
+msgstr "La demande de conditions d'authentification de l'endpoint personnalisé a échoué"
```

Comment:
```suggestion
msgstr "La demande d'endpoint personnalisé  de conditions d'authentification a échoué"
```

Reply:
@cg-tw There is a double space in your suggestion